### ISSUE TITLE
Feature Request: Consecutive Snackbars

### ISSUE THREAD

**Issue by christopher-howard** [jamesmfriedman/rmwc#547]
**Use Case**
I would like to use the Snackbar to display messages to the user as they click through my app, to provided a guided experience. This can happen when a new component mounts, or when they click a button for example. This requires being able to clear currently shown Snackbars when required, in order to show another, or as the current Snackbar may no longer be relevant after a user has performed another action. At the moment clearing is only provided by waiting for timeout to expire, or requiring user to click the action button on the Snackbar to close it. With more Snackbars, requiring the user to click the 'Got it' action on the Snackbar repeatedly to close each provides a cumbersome experience. 

**My Attempt**
I am currently trying to clear the queue to remove the previous Snackbar on the screen before I call queue.notify(...) to show a new one, as follows:

```
const queue = createSnackbarQueue();
const clear = () => {
  queue.messages.array = [];
};
// remove previous Snackbar (#1), display a new one (#2)
const onClick = () => {
  clear();
  queue.notify({ message: "I am Snackbar #2" });
};
```

This appears to work in removing the previous Snackbar and showing the new, but when using this method the newly shown Snackbar appears to receive a timeout that is equal to the remainder of the previous Snackbar's timeout. So if Snackbar #1 has a 10s timeout, and is on screen for 5s before the button is clicked and Snackbar #2 is shown, then even if Snackbar #2 was set to have a 10s timeout, it will disappear after 5s.

**Other**
It appears Material-UI provides this feature, though this may not be able to be used from anywhere in the app. See here for reference: https://material-ui.com/components/snackbars/

**Comment by jamesmfriedman** [comment]
This was added as an enhancement to RMWC V6, will go out in the next release.

**Comment by jamesmfriedman** [comment]
Fixed in 6.0.0
